0_intro_to_ml_using_jax.ipynb

```diff
@@ -0,0 +1,3758 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    f = lambda x: 6*x**4 - 9*x + 4
</pre><p>Why do we use a lambda function here instead of a regular function definition for f?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Reply:
<p>I guess that because it is close to the mathematical function f. </p>